spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/TaskLogsDocumentation.java

```diff
@@ -57,12 +73,45 @@ public void getLogsByTaskId() throws Exception {
 		this.mockMvc.perform(
 				get("/tasks/logs/"+taskDeploymentRepository.findTopByTaskDefinitionNameOrderByCreatedOnAsc(taskName)
 						.getTaskDeploymentId()).param("platformName", "default"))
-				.andDo(print())
 				.andExpect(status().isOk())
 				.andDo(this.documentationHandler.document(
 						requestParameters(
 								parameterWithName("platformName").description("The name of the platform the task is launched."))
 				));
 	}
+	@Test
+	public void getLogsByTaskExecutionId() throws Exception {
+		registerApp(ApplicationType.task, "timestamp", "1.2.0.RELEASE");
+		String taskName = "taskB";
+		documentation.dontDocument( () -> this.mockMvc.perform(
+						post("/tasks/definitions")
+								.param("name", taskName)
+								.param("definition", "timestamp --format='yyyy MM dd'"))
+				.andExpect(status().isOk()));
+		this.mockMvc.perform(
+						post("/tasks/executions")
+								.param("name", taskName))
+				.andExpect(status().isCreated());
+		TaskRepository taskRepository =
+				springDataflowServer.getWebApplicationContext().getBean(TaskRepository.class);
+		TaskDeploymentRepository taskDeploymentRepository =
+				springDataflowServer.getWebApplicationContext().getBean(TaskDeploymentRepository.class);
+		TaskExplorer taskExplorer = springDataflowServer.getWebApplicationContext().getBean(TaskExplorer.class);
+
+		Thread.sleep(30000);
```

Comment:
Ouch... 30secs?

Reply:
Sounds good.   30 seconds was what we used before.  I'll update both tests.